batch/file-ingest-sftp-cf/data/people.csv

```diff
@@ -0,0 +1,3 @@
+Jane,Doe
+John,Doe
+Joe,Doe
```

Comment:
Add \n

Reply:
this was intentional - IIRC adding an empty line causes an exception as the line mapper would have no values to split. there is a configuration option `linesToSkip` but applies to the top of the file https://docs.spring.io/spring-batch/trunk/reference/html/readersAndWriters.html#flatFileItemReader